### DISCUSSION TITLE
[Feat]: A services marketplace

### DISCUSSION THREAD

**Discussion by zillowez** [coollabsio/coolify#3134] [Feature Requests]
### Description

A marketplace (maybe a repo) for one click services.
Users can deploy or upload a new services from a website (e.g. market.coolify.io ) or from the same coolify ui.
Users can up vote and down vote new services.
Users can upload services by a web interface that ask for docker file and a docker compose file, and an environment variables, etc.
I know that's a lot of work, but it will be awesome.

### Minimal Reproduction (if possible, example repository)

A feature request

### Version

v4.0.0-beta.293

**Comment by YazeedAlKhalaf** [comment]
hey @ZilloweZ, I think the current solution is cool and easier to manage.
all things in one centralized place, easy to deploy too.

making a market place for people to upload stuff will be a lot of work, and we are devs, so i think github is a better place to handle those.

**Comment by zillowez** [comment]
Yeah, I'm planning this for the future, maybe when v4 releases
